example_case_folders/05_floris_wind_standin_and_electrolyzer/bash_script.sh

```diff
@@ -0,0 +1,19 @@
+# Example bash for running things locally
+# I just run these one at a t time
+
+# A lot of modules and conda stuff
+conda activate hercules
+
+# Delete old log files
+rm hercules_output_control.csv log_test_client.log loghercules_cl logfloris_cl
+
+# Set the helics port to use: 
+export HELICS_PORT=32000
+
+#make sure you use the same port number in the amr_input.inp and hercules_input_000.yaml files. 
+# Set up the helics broker
+
+helics_broker -t zmq  -f 2 --loglevel="debug" --local_port=$HELICS_PORT & 
+python3 hercules_runscript_CLcontrol.py hercules_input_000.yaml >> loghercules_cl 2>&1 &
+python3 floris_runscript.py amr_input.inp amr_standin_data-higher.csv >> logfloris_cl 2>&1
```

Comment:
I think that the `amr_standin_data-higher.csv` argument is not needed here? Or perhaps it is, but it doesn't appear in the repository, so I wasn't able to run the example with this line as is. However, it does run when I take out the `amr_standin_data-higher.csv` argument though (but I'm not sure if it's running the right thing... :) )

Reply:
Yep, I changed the name for this and didn't update it. It should be floris_standin_data.csv and is updated now!

Reply:
Thanks! While you're at it, do the output files need the `_cl` appended? This causes them to be missed by the .gitignore, so it might be best to name them `loghercules` and `logfloris`---or, update the .gitignore to ignore `loghercules*` and `logfloris*`, which might be a better general solution?

Reply:
The suggested updates to .gitignore are now in #93